railties/lib/rails/all.rb

```diff
@@ -13,6 +15,8 @@
   rails/test_unit/railtie
   sprockets/railtie
 ).each do |railtie|
-  require railtie
-rescue LoadError
+  begin
+    require railtie
+  rescue LoadError
```

Comment:
I'm curious about what is the meaning the rescuing LoadError.

The rescue was added at https://github.com/rails/rails/commit/dc677f7665e5ec74b5a313ba656bba19dc0f853d#diff-5729a03dfd2edf582f8f35a6bacda619R7.

Does this mean that ignoring LoadError even if loading any component is failed?